### PR TITLE
bip174: typo in some fixtures (using wrong magic prefix)

### PR DESCRIPTION
**Pull request by afk11**
`bin2hex("psbt\xff") = 70736274ff`

A few of the invalid PSBT cases have been serialized using the wrong magic, but only in their base64 encodings.

ping @achow101

### CODE REVIEW

File: bip-0174.mediawiki [hunk 1]
```diff
@@ -439,55 +439,55 @@ The following are invalid PSBTs:
...
+** Base64 String: <pre>cHNidP8BAFUCAAAAASeaIyOl37UfxF8iD6WLD8E+HjNCeSqF1+Ns1jM7XLw5AAAAAAD/////AaBa6gsAAAAAGXapFP/pwAYQl8w7Y28ssEYPpPxCfStFiKwAAAAAAAEBIJVe6gsAAAAAF6kUY0UgD2jRieGtwN8cTRbqjxTA+uHIQIDsTQcy6doO2r08SOM1ul+cWfVafrEfx5I1HVBhENVvUYwQwIgBCS1jv+qppThVZ6lyTu/1KiQZCJAVc3wcLZ3FGlELQcCH1yOsP6mUW1guKyzOtZO3mDoeFv7OqlLmb34YVHbmpoBAQQiACB3H9GK1FlmbdSfPVZOPbxC9MhHdONgraFoFqjtSI1WgQEFR1IhA7E0HMunaDtq9PEjjNbpfnFn1Wn6xH8eSNR1QYRDVb1GIQPeVdHh2sgF4/iljB+/m5TALz26r+En/vykmV8m+CCDvVKuIgYDsTQcy6doO2r08SOM1ul+cWfVafrEfx5I1HVBhENVvUYQtKa6ZwAAAIAAAACABAAAgCIGA95V0eHayAXj+KWMH7+blMAvPbqv4Sf+/KSZXyb4IIO9ELSmumcAAACAAAAAgAUAAIAAAA==</pre>
```
Comment (achow101) [inline bip-0174.mediawiki]: This doesn't appear to match the hex

Comment (afk11) [reply bip-0174.mediawiki]: Thanks, amended the commit

**Comment by luke-jr** [comment]
@achow101

**Comment by achow101** [comment]
ACK 2b7b95996a8e5f3f81d4a746969f9a123ba7b999

**Pull request merged**